packages/buildbucket-dart/tool/regenerate.sh

```diff
@@ -5,14 +5,24 @@
 
 set -ex
 
-mkdir -p lib/src/generated
+NO_CLONE=1
```

Comment:
Nit: seems a bool variable is enough.

Reply:
There are no booleans in bash unless something has changed. 0 is true and 1 is false.